noah/src/main/scala/edu/uci/ics/cloudberry/noah/news/news.properties

```diff
@@ -0,0 +1,4 @@
+#Thu Jul 14 15:59:24 PDT 2016
+filePath=/Users/KyleMa/Documents/workspace/cloudberry
+ts=1468536911354
+apiKey=d29db614-cfaa-456d-a78b-397632cf1dd1
```

Comment:
I think these `apiKey` should not be exposed publicly to the codebase.
You can just put the `input-your-webhose-key-here` to let us register and run it later.